web/src/settings.ts

```diff
@@ -115,6 +115,8 @@ export function build_page(): void {
         twenty_four_hour_time_values: settings_config.twenty_four_hour_time_values,
         general_settings: settings_config.all_notifications(user_settings).general_settings,
         notification_settings: settings_config.all_notifications(user_settings).settings,
+        stream_specific_notification_settings:
```

Comment:
Now that the object with details of checkbox is passed, and not only the setting names, we should rename this variable. Something like `custom_stream_specific_notification_settings`.